kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -525,22 +538,104 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
       }
     }
   }
+
+  private def handleUploadingFiles(
+      batchId: String,
+      request: BatchRequest,
+      resourceFileInputStream: InputStream,
+      resourceFileName: String,
+      formDataMultiPartOpt: Option[FormDataMultiPart]): Option[JPath] = {
+    val uploadFileFolderPath = batchResourceUploadFolderPath(batchId)
+    handleUploadingResourceFile(
+      request,
+      resourceFileInputStream,
+      resourceFileName,
+      uploadFileFolderPath)
+    handleUploadingExtraResourcesFiles(request, formDataMultiPartOpt, uploadFileFolderPath)
+    Some(uploadFileFolderPath)
```

Comment:
cleanup the session upload path if exception thrown

Reply:
Sure. Fixed.